wasm/build_kotlin_library.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Should this file be renamed to `build_wasm_library.sh`?

Reply:
I think it's fine, the build is for kotlin, just like the one in wrapper/